docs: Token generation doc (#4804)
kyma-project/busola: docs/user/01-43-generate-token.md

Reviewer: ```suggestion
   - **Kind**: `ServiceAccount`
```

# Generating a Token for Busola Instances

> When connecting to a cluster from a Busola instance (running via `npm start` or Docker), you must provide a kubeconfig that contains a valid authentication token. The following steps show how to create a dedicated ServiceAccount and generate a kubeconfig with a token for it — entirely from the Busola UI.

Reviewer: ```suggestion
When connecting to a cluster from a local Busola instance (running via `npm start` or Docker), you must provide a kubeconfig that contains a valid authentication token. The following steps show how to create a dedicated ServiceAccount and generate a kubeconfig with a token for it — entirely from the Busola UI.
```

Reviewer: So it means that without the token, a local Busila instance doesn't work?

> [!NOTE]
> You need an existing cluster connection in Busola to complete these steps. Use your current kubeconfig to connect first, then follow the steps below to create a dedicated token for future logins.

1. In your namespace, go to **Configuration** > **Service Accounts**.

2. Select **Create**, fill in the name (for example, `busola-dev`), and choose **Create**.

> 3. To grant the ServiceAccount access to cluster resources, in **Cluster Overview** go to **Configuration** > **Cluster Role Bindings**.

Reviewer: ```suggestion
3. To grant the ServiceAccount access to cluster resources, in **Cluster Overview**, go to **Configuration** > **Cluster Role Bindings**.
```

4. Open the `cluster-admin` role binding, and choose **Edit**.

   > [!NOTE]

>    > The `cluster-admin` role grants full access to all cluster resources. For local development this is typically convenient, but in shared or sensitive environments consider using a more restrictive ClusterRole.

Reviewer: ```suggestion
   > The `cluster-admin` role grants full access to all cluster resources. For local development, this is typically convenient, but in shared or sensitive environments, consider using a more restrictive ClusterRole.
```

> 5. In **Subjects** section, choose **Add Subject**, and in the newly created Subject, provide the following information:

Reviewer: ```suggestion
5. In the **Subjects** section, choose **Add Subject**, and in the newly created Subject, provide the following information:
```

>    - **Kind**: Service Account

Reviewer: Is Service Account the exact value you must type in? Isn't code font needed here?
- Service Account,
- ServiceAccount,
- `Service Account`,
- or `ServiceAccount`?

Author: It's sth you choose from the dropdown, and it's ServiceAccount :) 

<img width="590" height="246" alt="Image" src="https://github.com/user-attachments/assets/93a5711c-bb26-45ed-81e9-9a512f766542" />

# Generating a Token for Busola Instances

> When connecting to a cluster from a Busola instance (running via `npm start` or Docker), you must provide a kubeconfig that contains a valid authentication token. This is needed, for example, when you already have a remote Kyma Dashboard connected to your cluster and want to use a local Busola instance instead. The following steps show how to create a dedicated ServiceAccount and generate a kubeconfig with a token for it — entirely from the Busola UI.

Reviewer: ```suggestion
You have an existing cluster remotely connected to Kyma dashboard. At the same time, you want to connect to the cluster from a local Busola (Kyma dashboard) instance that you run using `npm start` or Docker. To enable the local connection, you must create a dedicated ServiceAccount and generate a kubeconfig with a token for it.
```
